frontend/public/locales/ru/translation.json

```diff
@@ -32,5 +32,36 @@
         "submit_filter": "Применить",
         "add_leader": "Добавить лидера",
         "add_project": "Добавить проект"
+    },
+
+    "errors": {
+        "page_500": {
+            "title": "Internal Server Error (500)",
+            "text": "Sorry, something went wrong with server.\nTry to reload site"
```

Comment:
Vot eto russky yezik!

Reply:
Yes, I'm russkiy
I think we can write translations later